server/views/pages/overview/partials/goalsTab/goalsTabContents.njk

```diff
@@ -51,7 +51,25 @@
             <p class="govuk-body" data-qa="no-in-progress-goals-message">{{ prisonerSummary.firstName }} {{ prisonerSummary.lastName }} has no goals in progress.</p>
           {% endif %}
         {% endset %}
-
+        {% if featureToggles.completedGoalsEnabled %} 
+          {% set completedGoalsHtml %}
+            <h2 class="govuk-heading-m">Completed goals</h2>
+            {% if completedGoals.length > 0 %}
+              {% for goal in completedGoals | sort(attribute = 'updatedAt', reverse = true) %}
```

Comment:
I think the sort order is wrong, not just on this tab but also the in-progress and archived tabs. Aliki has written a couple of tickets to address the in-progress and archived tabs.
Dont worry about fixing it on the in-progress and archived tabs (they are effectively a production bug so we'll need to get a fix out for those long before this feature is enabled in prod), but for this line I think we want to change the sort order to being sorted on `targetCompletionDate` . I'm not sure about the order though, whether its newest first or oldest first - we'll need to check with Aliki